test/largescale/tomo/astra_slow_test.py

```diff
@@ -0,0 +1,142 @@
+        geom = odl.tomo.Parallel3dGeometry(angle_intvl, dparams, agrid, dgrid,
+                                           axis=[1, 0, 0])
+
+        # X-ray transform
+        return odl.tomo.DiscreteXrayTransform(discr_reco_space, geom,
+                                              backend='astra_' + version)
+
+    elif geom == 'cone2d':
+        # Discrete reconstruction space
+        discr_reco_space = odl.uniform_discr([-20, -20],
+                                             [20, 20],
+                                             [100, 100], dtype='float32')
+
+        # Geometry
+        angle_intvl = odl.Interval(0, 2 * np.pi)
+        dparams = odl.Interval(-30, 30)
+        agrid = odl.uniform_sampling(angle_intvl, 200)
+        dgrid = odl.uniform_sampling(dparams, 200)
+        geom = odl.tomo.FanFlatGeometry(angle_intvl, dparams,
+                                        src_radius=200,
+                                        det_radius=100,
+                                        agrid=agrid, dgrid=dgrid)
+
+        # X-ray transform
+        return odl.tomo.DiscreteXrayTransform(discr_reco_space, geom,
+                                              backend='astra_' + version)
+
+    elif geom == 'cone3d':
+        # Discrete reconstruction space
+        discr_reco_space = odl.uniform_discr([-20, -20, -20],
+                                             [20, 20, 20],
+                                             [100, 100, 100], dtype='float32')
+
+        # Geometry
+        angle_intvl = odl.Interval(0, 2 * np.pi)
+        dparams = odl.Rectangle([-30, -30], [30, 30])
+        agrid = odl.uniform_sampling(angle_intvl, 200)
+        dgrid = odl.uniform_sampling(dparams, [200, 200])
+        geom = odl.tomo.CircularConeFlatGeometry(angle_intvl, dparams,
+                                                 src_radius=200,
+                                                 det_radius=100,
+                                                 agrid=agrid, dgrid=dgrid,
+                                                 axis=[1, 0, 0])
+
+        # X-ray transform
+        return odl.tomo.DiscreteXrayTransform(discr_reco_space, geom,
+                                              backend='astra_' + version)
+
+    else:
+        raise ValueError('param not valid')
+
+
+def test_reconstruction(projector):
+    """Test discrete X-ray transform using ASTRA for reconstruction."""
+
+    # Create shepp-logan phantom
+    vol = odl.util.shepp_logan(projector.domain, modified=True)
+
+    # Project data
+    projections = projector(vol)
+
+    # Calculate operator norm for landweber
+    op_norm_est_squared = projector.adjoint(projections).norm() / vol.norm()
+    omega = 0.5 / op_norm_est_squared
+
+    # Reconstruct using ODL
+    recon = projector.domain.zero()
+    odl.solvers.landweber(projector, recon, projections, niter=100,
+                          omega=omega)
+
+    assert recon.dist(vol) < vol.norm() / 3.0
```

Comment:
Its a number larger than 1, i.e. the error is somehow decreased. 3.0 seems to be about what we get to so I picked that. This is mostly to see if it somehow becomes worse. Perhaps add a comment.